### DISCUSSION TITLE
你好，2022 | Chaojie's Home✍️

### DISCUSSION THREAD

**Discussion by giscus** [ischaojie/blog#164] [Announcements]
# 你好，2022 | Chaojie's Home✍️

  Hi, I'm chaojie([潮戒](/about/)). I'm working at [Douban](https://en.wikipedia.org/wiki/Douban) as Python engineer.

  I'm interested in programming, movies, design and some hardware stuffs. Sometimes have unrealistic fantasies.

  I have a dog🐶 named [π](https://pai.dog/).

  “愿你自己有充分的忍耐去担当，有充分单纯的心去信仰。” ------ 里尔克

  

https://chaojie.fun/posts/hello_2022_and_automata/

**Comment by ischaojie** [comment]
test giscus
